pyaedt/application/design_solutions.py

```diff
@@ -542,17 +534,88 @@ def __init__(self, odesign, design_type, aedt_version):
         self._composite = None
         self._hybrid = None
 
+    @property
+    def solution_type(self):
+        """Get/Set the Solution Type of the active Design."""
+        if self._odesign:
+            try:
+                self._solution_type = self._odesign.GetSolutionType()
+                if "Modal" in self._solution_type:
+                    self._solution_type = "Modal"
+                elif "Terminal" in self._solution_type:
+                    self._solution_type = "Terminal"
+            except:
+                self._solution_type = solutions_defaults[self._design_type]
+        elif self._solution_type is None:
```

Comment:
@maxcapodi78 I am not sure if it is a `if` or `elif` here because we are not testing the same value. But I think what you did also works.
```suggestion
        if self._solution_type is None:
```

Reply:
it is intended.